integration-tests/jpa-postgresql-vault/pom.xml

```diff
@@ -0,0 +1,268 @@
+                </property>
+            </activation>
+            <build>
+                <plugins>
+                    <plugin>
+                        <artifactId>maven-surefire-plugin</artifactId>
+                        <configuration>
+                            <skip>false</skip>
+                            <trimStackTrace>false</trimStackTrace>
+                            <systemProperties>
+                                <!-- see TLS availability in native mode https://github.com/quarkusio/quarkus/issues/3797 -->
+                                <!-- to activate tls, set vault-test-extension.use-tls=true and install /opt/graalvm -->
+                                <javax.net.ssl.trustStore>${project.basedir}/../../extensions/vault/test/src/main/resources/vaultTrustStore</javax.net.ssl.trustStore>
+                                <vault-test-extension.use-tls>false</vault-test-extension.use-tls>
+                            </systemProperties>
+                        </configuration>
+                    </plugin>
+                    <plugin>
+                        <artifactId>maven-failsafe-plugin</artifactId>
+                        <configuration>
+                            <skip>false</skip>
+                            <trimStackTrace>false</trimStackTrace>
+                            <systemProperties>
+                                <!-- see TLS availability in native mode https://github.com/quarkusio/quarkus/issues/3797 -->
+                                <!-- to activate tls, set vault-test-extension.use-tls=true and install /opt/graalvm -->
+                                <javax.net.ssl.trustStore>${project.basedir}/../../extensions/vault/test/src/main/resources/vaultTrustStore</javax.net.ssl.trustStore>
+                                <vault-test-extension.use-tls>false</vault-test-extension.use-tls>
+                            </systemProperties>
+                        </configuration>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+    
+        <profile>
+            <id>native-image</id>
+            <activation>
+                <property>
+                    <name>native</name>
+                </property>
+            </activation>
+            <build>
+                <plugins>
+                    <plugin>
+                        <artifactId>maven-surefire-plugin</artifactId>
+                        <configuration>
+                            <trimStackTrace>false</trimStackTrace>
+                            <systemProperties>
+                                <!-- see TLS availability in native mode https://github.com/quarkusio/quarkus/issues/3797 -->
+                                <!-- to activate tls, set vault-test-extension.use-tls=true and install /opt/graalvm -->
+                                <javax.net.ssl.trustStore>${project.basedir}/../../extensions/vault/test/src/main/resources/vaultTrustStore</javax.net.ssl.trustStore>
```

Comment:
I think I would have copied the truststore in the modules where it's needed (there are a couples). I'm not a big fan of hidden cross-modules dependencies.

Reply:
done